evm/src/cpu/syscalls.rs

```diff
@@ -0,0 +1,110 @@
+//! Handle instructions that are implemented in terms of system calls.
+//!
+//! These are usually the ones that are too complicated to implement in one CPU table row.
+
+use once_cell::sync::Lazy;
+use plonky2::field::extension::Extendable;
+use plonky2::field::packed::PackedField;
+use plonky2::field::types::Field;
+use plonky2::hash::hash_types::RichField;
+use plonky2::iop::ext_target::ExtensionTarget;
+
+use crate::constraint_consumer::{ConstraintConsumer, RecursiveConstraintConsumer};
+use crate::cpu::columns::{CpuColumnsView, COL_MAP};
+use crate::cpu::kernel::aggregator::KERNEL;
+
+const NUM_SYSCALLS: usize = 2;
+
+fn make_syscall_list() -> [(usize, usize); NUM_SYSCALLS] {
+    let kernel = Lazy::force(&KERNEL);
```

Comment:
Did you want to make these explicit? Typically we use `KERNEL` etc directly and let its `Deref` do this